src/locale/dim.json

```diff
@@ -491,6 +491,7 @@
     "SizeItem": "Item size",
     "SortName": "name",
     "SortPrimary": "primary stat",
+    "SortBasePower": "base power (destiny 2 only)",
```

Comment:
If you do any of @kyleshay propositions, dont forget to remove the `(destiny 2 only)`